wazo_webhookd/database/models.py

```diff
@@ -60,6 +64,24 @@ class SubscriptionMetadatum(Base):
     value = Column(Text())
 
 
+class SubscriptionHookLog(Base):
+
+    __tablename__ = 'webhookd_subscription_hook_log'
+
+    uuid = Column(String(38), primary_key=True)
```

Comment:
Fixed, I copy/pasting previous line.

Any reason, that's all uuid are not stored with the uuid postgresql type ?

Reply:
When we started using UUID we did not know about the UUID type in Postgresql.
I don't know if we started before that type was added to Postgresql or if we just missed it.
I would be very pleased to see this changed. webhookd could be a good candidate to start that since it does not have too many tables yet.